### PR TITLE
Story/ba 510 rm dc

### PR DESCRIPTION
**Pull request by abnepc**
## Jira Story
[BA-510](https://epcpower.atlassian.net/browse/BA-510)

## About
Update Poetry version from 1.1.15 to 1.5.1.

## Associated Pull Requests
N/A

## Validation

[BA-510]: https://epcpower.atlassian.net/browse/BA-510?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pyproject.toml
Comment (gordon-epc) [inline pyproject.toml:1]: Add this comment before `version`:

```
# The version variable below is set to the generic build placeholder "0.0.0".
# (In other words: leave alone!) During CI, poetry dynamic versioning
# modifies the __version__ variable in the applicable __init__.py file.
```

Comment (gordon-epc) [inline pyproject.toml:1]: Remove this line:
`poetry-dynamic-versioning = "^0.13.1"`

File: pyproject.toml [hunk 1]
```diff
@@ -85,11 +81,10 @@ exclude = './.venv'
...
 requires = [
-    "alqtendpy",
     "setuptools",
     "wheel",
     "poetry-core>=1.0.0",
-    "poetry-dynamic-versioning",
+    "poetry-dynamic-versioning>=1.0.0,<2.0.0",
 
 ]
 build-backend = "poetry.core.masonry.api"
```
Comment (gordon-epc) [inline pyproject.toml]:
```suggestion
build-backend = "poetry_dynamic_versioning.backend"
```

File: .github/workflows/ci.yml
Comment (gordon-epc) [inline .github/workflows/ci.yml:1]: Add the `ChefBuildError` workaround. We may as well do it now, as I see it affecting more and more repos.

**Review by gordon-epc — CHANGES_REQUESTED** [review]

File: pyproject.toml
Comment (abnepc) [reply pyproject.toml:1]: [40a7e55](https://github.com/epcpower/pm/pull/97/commits/40a7e55afd808adf569a0b9ee607e5a7878f9895)

File: pyproject.toml (hunk 1 above)
Comment (abnepc) [reply pyproject.toml:92]: [40a7e55](https://github.com/epcpower/pm/pull/97/commits/40a7e55afd808adf569a0b9ee607e5a7878f9895)

File: pyproject.toml
Comment (abnepc) [reply pyproject.toml:1]: [40a7e55](https://github.com/epcpower/pm/pull/97/commits/40a7e55afd808adf569a0b9ee607e5a7878f9895)

File: .github/workflows/ci.yml
Comment (abnepc) [reply .github/workflows/ci.yml:1]: [f92c6dc](https://github.com/epcpower/pm/pull/97/commits/f92c6dc535415bf9d09a5e007e39e49dc7863c07)

File: pyproject.toml (hunk 1 above)
Comment (abnepc) [reply pyproject.toml]: [79206fb](https://github.com/epcpower/pm/pull/97/commits/79206fba3b40e133b352add0207f56af00873f4f)

**Review by gordon-epc — APPROVED** [review]

**Pull request merged**